server/main.lua

```diff
@@ -39,12 +39,35 @@ if Config.UseWolfknightRadar == true then
 	AddEventHandler("wk:onPlateScanned", function(cam, plate, index)
 		local src = source
 		local Player = QBCore.Functions.GetPlayer(src)
-		local bolo = GetBoloStatus(plate)
+		local PlayerData = GetPlayerData(src)
+		local vehicleowner = GetVehicleOwner(plate)
+		local bolo, title, boloid = GetBoloStatus(plate, title, boloid)
+		local warrant, owner, incidentid = GetWarrantStatus(plate, owner, incidentid)
+		local driverslicense = PlayerData.metadata['licences'].driver
+		local driverunlicensed = nil
+
 		if bolo == true then
-			TriggerClientEvent("wk:togglePlateLock", src, cam, true, bolo)
+			TriggerClientEvent('QBCore:Notify', src, 'BOLO ID: '..boloid..' | Title: '..title..' | Registered Owner: '..vehicleowner..' | Plate: '..plate, 'error', Config.WolfknightNotifyTime)
+		end
+		if warrant == true then
+			TriggerClientEvent('QBCore:Notify', src, 'WANTED - INCIDENT ID: '..incidentid..' | Registered Owner: '..owner..' | Plate: '..plate, 'error', Config.WolfknightNotifyTime)
+		end
+
+		if driverslicense == false then

```

Comment:
I would recommend adding a config value to enable/disable this check. Some people may not want to check this on platescan. 

`if Config.PlateScanForDriversLicense and not driversLicense then ...`

Reply:
I was mainly referring to the checking of the driverslicense as something that should be toggleable via config. 

For example, in my server, I would use this to scan for bolo plates and warrants. I may not want to scan for whether the registered owner of a vehicle had a valid driverslicense tho, but thats just me 🤷